package.json

```diff
@@ -28,6 +28,7 @@
     "react-dom": "^16.13.1",
     "react-image-picker": "^1.1.1",
     "react-pdf": "^5.0.0",
+    "react-pin-input": "^1.0.1",
     "react-plotly.js": "^2.5.0",
```

Comment:
react pin input (https://www.npmjs.com/package/react-pin-input) is a small package that is great for handling pin validation - it cut down the amount of code needed to validate - you can compare this pin validation against the other one earlier in the user flow to see the difference.